lte/gateway/c/oai/tasks/s1ap/s1ap_state_manager.cpp

```diff
@@ -163,13 +165,22 @@ void S1apStateManager::clear_s1ap_imsi_map() {
   hashtable_uint64_ts_destroy(s1ap_imsi_map_->mme_ue_id_imsi_htbl);
 
   free_wrapper((void**) &s1ap_imsi_map_);
+
+  if(persist_state_enabled) {
+    std::vector<std::string> keys_to_del;
+    keys_to_del.emplace_back(S1AP_IMSI_MAP_TABLE_NAME);
+    redis_client->clear_keys(keys_to_del);
+  }
 }
 
 s1ap_imsi_map_t* S1apStateManager::get_s1ap_imsi_map() {
   return s1ap_imsi_map_;
 }
 
 void S1apStateManager::put_s1ap_imsi_map() {
```

Comment:
seems like this function is committing to the db, shall we change the name to `write_s1ap_imsi_map_to_db`?